src/proto/play.proto

```diff
@@ -4,29 +4,22 @@ package TbotsProto;
 
 enum PlayName
 {
-    UseAiSelection                     = 1;
-    ShootOrChipPlay                    = 2;
-    KickoffFriendlyPlay                = 3;
-    OffensePlay                        = 4;
-    CornerKickPlay                     = 5;
-    ShootOrPassPlay                    = 6;
-    BallPlacementPlay                  = 7;
-    HaltPlay                           = 8;
-    EnemyFreeKickPlay                  = 9;
-    ExamplePlay                        = 10;
-    DefensePlay                        = 11;
-    EnemyBallPlacementPlay             = 12;
-    PenaltyKickEnemyPlay               = 13;
-    PenaltyKickPlay                    = 14;
-    FreeKickPlay                       = 15;
-    CreaseDefensePlay                  = 16;
-    StopPlay                           = 17;
-    KickoffEnemyPlay                   = 18;
-    AssignedTacticsPlay                = 19;
-    ScoringWithStaticDefendersPlay     = 20;
-    DribblingParcourPlay               = 21;
-    ScoringFromContestedPossessionPlay = 22;
-    PassEndurancePlay                  = 23;
+    reserved 2, 5, 6, 10, 16;
```

Comment:
keep `ExamplePlay`. `ExamplePlay` is quite convenient during field testing

Reply:
done